Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -124,6 +126,8 @@ public void Initialize(TelemetryConfiguration configuration)
                             // Net40 only supports runtime instrumentation
                             // Net45 supports either but not both to avoid duplication
                             this.InitializeForRuntimeInstrumentationOrFramework();
+#else
+                            this.sqlClientDiagnosticSourceListener = new SqlClientDiagnosticSourceListener(configuration);
```

Comment:
We have previously discovered that even if application targets .NET Full Framework, it may install some package that targets netstandard1.*. That will bring the whole .NET Core stack into such application. In this case SqlClient may be .NET Core SqlClient.

For HTTP we decided to enable listener on NET45 as well (see `httpCoreDiagnosticSourceListener` initialization). It costs nothing to have no-op listener.